packages/nodes-base/nodes/RabbitMQ/GenericFunctions.ts

```diff
@@ -2,12 +2,6 @@ import { IDataObject, IExecuteFunctions, ITriggerFunctions, sleep } from 'n8n-wo
 
 import * as amqplib from 'amqplib';
 
-declare module 'amqplib' {
```

Comment:
[pushed this change upstream](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63506#event-8068208688)